docs/resources/project_rolling_release.md

```diff
@@ -0,0 +1,76 @@
+---
```

Comment:
add an examples/resources/vercel_project_rolling_release/import.sh

Reply:
added in b9dd027792515f09998ec0b714c8a926881021f2